services/src/main/java/org/keycloak/authentication/authenticators/broker/IdpReviewProfileAuthenticator.java

```diff
@@ -46,6 +46,10 @@
 public class IdpReviewProfileAuthenticator extends AbstractIdpAuthenticator {
 
     private static final Logger logger = Logger.getLogger(IdpReviewProfileAuthenticator.class);
+    private static final String FIRST_NAME = "lastName";
```

Comment:
I am not sure why the constants are needed here? Also not sure why FIRST_NAME is set to the "lastName"? It seems this is typo? Is it possible to remove all changes in IdpReviewProfileAuthenticator?

Reply:
Yes, since we didn't rename anything in the end, we can just remove the changes (Done).

They were made originally because the form data fields are independent of the UserModel field names (or at least it seemed that way). I still think that maybe you'd want to have independent constants because those two areas could change independently, but it's not necessary, so I reverted the change.